npm/install.js

```diff
@@ -0,0 +1,82 @@
+const fs = require("fs");
+const https = require("https");
+const path = require("path");
+
+const packageJson = require("./package.json");
+const { version, name, repository } = packageJson;
```

Comment:
Looks like version is being read in from the package json. So the version of the NPM package itself correlates directly to the version of the upstream software?

Reply:
Yes. The release system will bump the version when a release is made and the bump is committed.

Reply:
Interesting: looks like I need to go read up on that action! So it'll automagically bump the `package.json` version for us?

Reply:
Yes. Here's a demo bump with the CHANGELOG.
https://github.com/Anush008/xdg/commit/d28cfd9525f9b98cbced3d84b38b809a36e836e1